src/commit.ts

```diff
@@ -40,7 +41,8 @@ export default async ({
         await exec('git', ['add', '-A'], options)
 
         try {
-            await exec('git', ['commit', '-v', '-m', `${MESSAGE}`], options)
+            const trailerParams = skipCiMessageLocation == 'trailer' ? ['--trailer', 'skip-checks: true'] : []
```

Comment:
very minor(consistency): looking at the rest of the code, it seems that most of the logic for deriving values based on the inputs is in `main.ts` (e.g. `tagMsg`). For consistency, I wonder if it would make sense to pass in a `trailer` param instead and just do:

```suggestion
            const trailerParams = trailer && ['--trailer', trailer]
```

also, my previous suggestion to use a ternary was overly verbose (simplified above using `&&`)